stardist/models/model3d.py

```diff
@@ -67,30 +75,36 @@ def __getitem__(self, i):
         if X.ndim == 4: # input image has no channel axis
             X = np.expand_dims(X,-1)
 
-        tmp = [edt_prob(lbl, anisotropy=self.anisotropy) for lbl in Y]
+        tmp = np.stack([edt_prob(lbl, anisotropy=self.anisotropy) for lbl in Y])
```

Comment:
I think the reason for `self.out_edt_prob` and `self.out_star_dist3D` was that `np.stack` was the time bottleneck. This advantage is now gone...

Reply:
Should be fixed